source/renderer/app/api/wallets/types.js

```diff
@@ -71,7 +71,11 @@ export type WalletUnit = WalletUnits.LOVELACE | WalletUnits.ADA;
 
 export type AdaWallets = Array<AdaWallet>;
 
-export type SyncStateStatus = 'ready' | 'restoring' | 'syncing';
+export type SyncStateStatus =
```

Comment:
@daniloprates @nikolaglumac Don't we need to define constants for static string literals and use that here?
Similar to `WalletUnits`
```
export const WalletUnits: {
  ADA: WalletUnit,
  LOVELACE: WalletUnit,
} = {
  ADA: 'ada',
  LOVELACE: 'lovelace',
};
```

Reply:
Hey @yakovkaravelov the PR is still in WIP, but I agree and was going to work on that. Thanks!